### ISSUE TITLE
Custom codes overwritten when config file submitted via API

### ISSUE THREAD

**Issue by sw104** [fontello/fontello#734]
I've written a simple Perl script to create a new session based on my config file using the developers API. The session is created successfully with the correct symbols, however the codes specified in the config file are overwritten in the session by the defaults.

Manually uploading the same config file to the website to create a session does not suffer from this issue.

Here is a minimal version of the Perl code I am using:
```
#!/usr/bin/perl

use strict;
use warnings;

use LWP::UserAgent;
use FindBin;

my $ua = LWP::UserAgent->new;
my $url = 'https://fontello.com/';

# Path to config file
my $config = "$FindBin::Bin/config.json";

# Submit post request with config file
my $resp = $ua->post($url,
  Content_Type  => 'form-data',
  Content       => [
    'config'  => [ $config ],
  ],
);

print 'Response: ' . $resp->decoded_content . "\n";
exit(0);
```

**Screenshots**

The codes used in my config file are overwritten in the session as you can see here:
![image](https://user-images.githubusercontent.com/8680919/111034584-4eb7a980-840e-11eb-8850-218ea5843606.png)

Importing the same config file directly via the website keeps my customised codes:
![image](https://user-images.githubusercontent.com/8680919/111034645-8888b000-840e-11eb-839a-dac931774126.png)

**Config file**

```
{
  "name": "",
  "css_prefix_text": "icon-",
  "css_use_suffix": false,
  "hinting": true,
  "units_per_em": 1000,
  "ascent": 850,
  "glyphs": [
    {
      "uid": "d35a1d35efeb784d1dc9ac18b9b6c2b6",
      "css": "pencil",
      "code": 9998,
      "src": "fontawesome"
    },
    {
      "uid": "9daa1fdf0838118518a7e22715e83abc",
      "css": "file-pdf",
      "code": 128462,
      "src": "fontawesome"
    },
    {
      "uid": "57a0ac800df728aad61a7cf9e12f5fef",
      "css": "flag",
      "code": 9873,
      "src": "fontawesome"
    }
  ]
}
```

**OS**

Linux (Debian).

**Comment by rlidwka** [comment]
This looks like a duplicate of https://github.com/fontello/fontello/issues/743, fixed in https://github.com/fontello/fontello/commit/e2689251690c53774cde12a843f7d1c6c028ad12.

Can you check if it works now?

**Comment by rlidwka** [comment]
Gonna close this, please reopen if the bug somehow still exists.

**Comment by sw104** [comment]
> This looks like the same issue as #743, fixed in [e268925](https://github.com/fontello/fontello/commit/e2689251690c53774cde12a843f7d1c6c028ad12).
> 
> Can you check if it works now?

Yes, this is now fixed. Thanks! :smile:
